features/netsocket/Socket.h

```diff
@@ -111,9 +111,9 @@ class Socket {
 
     /** Send a message on a socket.
      *
-     * The sendto() function shall send a message through a connection-mode or connectionless-mode socket.
-     * If the socket is connectionless-mode, the message shall be sent to the address specified.
-     * If the socket is connection-mode, address shall be ignored.
+     * The sendto() function sends a message through a connection-mode or connectionless-mode socket.
+     * If the socket is in connectionless-mode, the message is sent to the address specified.
+     * If the socket is in connection-mode, address is ignored.
```

Comment:
@melwee01 
Actually, this is now incorrect.

TCPSocket is a connected socket.
UDPSocket is a connectionsless socket.

It is a protocol specific thing. Not something you can switch.

Therefore the `socket is in connection-mode` is a bit misleading. Socket is either connected, or connectionless.

This is how POSIX standard defines the same thing: http://pubs.opengroup.org/onlinepubs/9699919799/functions/send.html

They use wording `If the socket is a connectionless-mode socket,...`

Reply:
I'd either use the article such as in your example, or take out -mode

Reply:
Either way, it's up to you. I'm off to lunch :)

Reply:
I'll take out the `-mode`

Reply:
Actually.. that was causing more confusion.. so I'll change the `in` to `a`

```
     * If the socket is a connectionless-mode socket, the message is sent to the address specified.
     * If the socket is a connected-mode socket, address is ignored.
```